docs/src/concepts/psts.md

```diff
@@ -0,0 +1,57 @@
+# Overview
+
+---
+
+Profit Sharing Tokens (PSTs) are tokens issued by a community or protocol created on the permaweb. The protocol issuing a PST is often referred to as a ‘Profit Sharing Community’ (PSC).
+
```

Comment:
PSTs do not have to be issued by a PSC, they can be created as SWT or SmartWeaveTokens:

* balances object
* name string
* ticker string

Reply:
Will adjust, thanks